src/ast/mod.rs

```diff
@@ -4556,13 +4556,16 @@ pub enum FunctionArgOperator {
     Equals,
     /// function(arg1 => value1)
     RightArrow,
+    /// function(arg1 := value1)
+    Assignment,
```

Comment:
can we rename this to `DuckAssignment` just to be consistent with the tokenizer naming. 

```suggestion
    DuckAssignment,
```

Reply:
Of course, it's done!